utils/physics_objects.py

```diff
@@ -0,0 +1,198 @@
+import awkward as ak
```

Comment:
Can we keep in this file only what is needed?
Also, most of the code here is only valid for NanoAOD-like NTuples... Either it is made compatible for everyone and moved to `skimmer` maybe. If kept like this, I would move it to `analysis_configs` and call it `physics_objects_nano_aod.py` so that it's a limited scope and not in the area common for all users.

Reply:
moved in analysis_configs as physics_objects_nano_aod.py